test/extensions/Worker.Extensions.Http.AspNetCore.Tests/HttpResultAttributeExpectedTests.cs

```diff
@@ -347,6 +347,119 @@ public class MyOutputType
             await test.RunAsync();
         }
 
+        [Fact]
+        public async Task HttpResultAttribute_WhenReturnTypeIsWrappedInTask_Expected()
+        {
```

Comment:
Can we add some tests which ensure that we don't suggest code fix when its not needed? For example, what if there is another method in this class (which is not a trigger) but has Task<poco> as return type, something like below -

```csharp
Task<poco> fun() { }
```